### PR TITLE
[FEATURE] Added a new env var in order to set the agent conf with a Docker volume

### PR DESCRIPTION
**Pull request by lammoth**
I wrote a new feature to set the PostGIS sink conf with the data of a volume.

### CODE REVIEW

File: docker/cygnus-ngsi/cygnus-entrypoint.sh [hunk 1]
```diff
@@ -582,7 +582,18 @@ if [ "$CYGNUS_ORION_HOST" != "" ]; then
 fi
 
 # Check if POSTGIS ENV vars
-if [ "$CYGNUS_POSTGIS_HOST" != "" ]; then
+if [ "${CYGNUS_POSTGIS_ENABLE_VOLUME_CONFIG,,}" == "true" ]; then
+    AGENT_CONF_FILE=agent_postgis.conf
```
Comment (AlvaroVega) [inline docker/cygnus-ngsi/cygnus-entrypoint.sh]: What kind of config are in your agent_postigs.conf that are not in conf env vars? Which other conf files a re in your volume?

Comment (lammoth) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh]: Good question, an important note, the volume only mount files, so, we can mount the `agent_postgis.conf`, `name_mappings_postgis.conf` and `grouping_rules_postgis.conf`.

Comment (lammoth) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh]: This is an example extracted from the `docker-compose.yml`:

```yaml
iot-cygnus:
  container_name: iot-cygnus
  hostname: iot-cygnus
  image: telefonicaiot/fiware-cygnus:latest
  volumes:
    - ${DBPATH_POSTGIS_AGENT}:/opt/apache-flume/conf/agent_postgis.conf
    - ${DBPATH_POSTGIS_MAPPING}:/opt/apache-flume/conf/name_mappings_postgis.conf
environment:
    - CYGNUS_POSTGIS_ENABLE_VOLUME_CONFIG =true
...
```

Comment (fgalan) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh]: I'd suggest to use a name for the env var closer to what it actually does. For instance:

```
CYGNUS_POSTGIS_SKIP_CONF_GENERATION
```

From my point of view, the possibility of putting the configuration in a volume is *a consequence* of skipping configuration generation.

In addition, this env var should be documented at https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md, in the Postgis section.

Comment (lammoth) [reply docker/cygnus-ngsi/cygnus-entrypoint.sh]: Yep, definitely, your proposal is more accurate, the change is [here](https://github.com/telefonicaid/fiware-cygnus/pull/1657/commits/ab6eadab326b5cb1844d3cc263fd6f1b09a4fde0).

File: doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md [hunk 2]
```diff
@@ -307,6 +307,7 @@ As seen above, the default configuration distributed with the image is tied to c
     * The port for Postgis is `3306` but can be changed through the CYGNUS_POSTGIS_PORT environment variable.
     * The user for Postgis is `postgresql` but can be changed through the CYGNUS_POSTGIS_USER environment variable.
     * The pass for Postgis is `postgresql` but can be changed through the CYGNUS_POSTGIS_PASS environment variable.
+    * CYGNUS_POSTGIS_SKIP_CONF_GENERATION: true skips the generation of the conf files, typically this files will be got from a volume, false generate the conf files.
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md]: Maybe more precise:

false generate the conf files -> false autogenerate the conf files from the rest of environment variables.

Comment (lammoth) [reply doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md]: Ok, the change is [here](https://github.com/telefonicaid/fiware-cygnus/pull/1657/commits/61aef2004d42967c5293839c10bc355144f04747).

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**